docs/pages/getting-started/6-connecting-ehr.rst

```diff
@@ -0,0 +1,59 @@
+.. _connecting-crm:
+
+Getting Started on EHR integration
+##################################
+
+This getting started manual assumes the vendor and its clients (care organizations) are set up on the Nuts Network through :ref:`connecting-crm`.
+The next step is to integrate the vendor's electronic health record (EHR) with the Nuts node to execute Bolts.
+
+All APIs used in the following chapters are documented at the :ref:`API <nuts-node-api>` page.
+Open API Spec files are available for generating client code.
```

Comment:
Where can I find them online? (Maybe something that has to be added to the API page)